kubernetes/helm/collabora-online/templates/configmap.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: haproxy-configmap
+  namespace: {{ .Values.namespace.collabora }}
+data:
+  timeout-tunnel: 3600s
+  request-set-header: X-HAProxy-loolwsd %[url_param(WOPISrc)]
```

Comment:
We've just got a feedback from @juliushaertl that this HAProxy setup did not work well with full screen presentation, and he suggested a different setting. @lpranam could you please verify it?
* request-set-header is not needed.
* load-balance: balance url_param WOPISrc check_post

Reply:
yes, I just checked presentation is not loading.